### ISSUE TITLE
Abort in debugger

### ISSUE THREAD

**Issue by lheckemann** [NixOS/nix#10134]
**Describe the bug**

The debugger crashes under certain circumstances (unclear what these are, but I've been able to trigger this reliably):

```
$ nix run github:nixos/nix/2.20.4 -- build github:lheckemann/nixpkgs/broken-please-ignore-unless-youre-debugging-nix#nixosTests.hedgedoc --debugger --ignore-try
error: The option `nodes.hedgedocPostgresWithTCPSocket.services.hedgedoc.environmentFile' does not exist. Definition values:
       - In `makeTest parameters': <derivation hedgedoc-env>


Starting REPL to allow you to inspect the current state of the evaluator.

evaluating derivation 'github:lheckemann/nixpkgs/broken-please-ignore-unless-youre-debugging-nix#nixosTests.hedgedoc'Aborted (core dumped)

```

**Steps To Reproduce**

1. `nix run github:nixos/nix/2.20.4 -- build github:lheckemann/nixpkgs/broken-please-ignore-unless-youre-debugging-nix#nixosTests.hedgedoc --debugger --ignore-try`

**Expected behavior**

Debugger opens and allows inspecting the evaluation state.

**`nix-env --version` output**

2.20.4

**Priorities**

Add :+1: to [issues you find important](https://github.com/NixOS/nix/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).

**Comment by 9999years** [comment]
I can't reproduce this crash on the latest `master`, can you confirm?

**Comment by lheckemann** [comment]
Indeed, it's fixed in 5ccb06ee1b4c757ff4ca0aa6eac15d5656f7774c.
